### PR TITLE
Add filtering by process id, thread id, or process name

### LINKED ISSUE #1: Add filtering by process id or thread id
Ideally with the same `--where` syntax as for other payload fields. If possible, enrich the events with the process name as well, so we can filter over those too.

### PR DESCRIPTION
**Pull request by Pavel-Durov**
Added support for event filtering using --where verb, example:
 
--kernel Process --event --where ProcessName="<your-process-name>" 
 --kernel Process --event --where ThreadId="<your-thread-Id>"
 --kernel Process --event --where ProcessId="<your-process-Id>"
 
However, the filters doesn't depend one on another, for exmple:
 --kernel Process --event --where ProcessId="<your-process-Id>" ThreadId="<threadId>"
will generate the same output as 
 --kernel Process --event --where ProcessId="<your-process-Id>"
* Because it will match ProcessId is both filters.

Does this functionality required?

____

Updated: resolves #1.

### CODE REVIEW

**Comment by goldshtn** [comment]
Are the quotes required? I.e. can I do `--where ProcessId=4428`? Regarding whether filters are any-of or all-of, let's stay with the current functionality, but it can be a potentially interesting addition.

**Comment by goldshtn** [comment]
Also would be nice to add an example showing the new functionality (e.g. filtering by process name).

**Comment by Pavel-Durov** [comment]
Quotes not required `--where ProcessId=4428 `is identical to `--where ProcessId="4428"` (+examples added)

File: README.md [hunk 1]
```diff
@@ -210,6 +210,9 @@ Examples:
   etrace --file trace.etl --stats
   etrace --clr GC --event GC/Start --field PID,TID,Reason[12],Type
   etrace --kernel Process --event Process/Start --where ImageFileName=myapp
+  etrace --kernel Process --event --where ProcessName=myProcessName
+  etrace --kernel Process --event --where Thread/Start ProcessId=4
```
Comment (goldshtn) [inline README.md]: @Pavel-Durov I don't think is the right order, is it? Also, why put `--event` there if you're not including any events to filter?

File: etrace/Options.cs [hunk 2]
```diff
@@ -87,6 +87,9 @@ public string Usage()
             help.AddPostOptionsLine("  etrace --file trace.etl --stats");
             help.AddPostOptionsLine("  etrace --clr GC --event GC/Start --field PID,TID,Reason[12],Type");
             help.AddPostOptionsLine("  etrace --kernel Process --event Process/Start --where ImageFileName=myapp");
+            help.AddPostOptionsLine("  etrace --kernel Process --event --where ProcessId=4");
+            help.AddPostOptionsLine("  etrace --kernel Process --event --where ProcessName=myProcessName");
```
Comment (goldshtn) [inline etrace/Options.cs]: Same comment here.

**Review by goldshtn — CHANGES_REQUESTED** [review]

File: etrace/Options.cs (hunk 2 above)
Comment (Pavel-Durov) [reply etrace/Options.cs]: fixed

File: README.md (hunk 1 above)
Comment (Pavel-Durov) [reply README.md]: fixed

**Pull request merged**